src/goto-symex/symex_function_call.cpp

```diff
@@ -200,13 +200,14 @@ void goto_symext::symex_function_call_symbol(
 {
   code_function_callt code = original_code;
 
+  symex_live_let_variablest live_let_variables(*this, state);
```

Comment:
Hmm, when exactly do we (not) need a new such context? Can we possibly reuse frames for this purpose? It's another bit of state to track and the interplay with branching I'm not entirely sure of.